libs/utils/utils.js

```diff
@@ -1077,7 +1078,7 @@ export async function loadArea(area = document) {
     scrollToHashedElement(currentHash);
   }
 
-  if (isDoc) await documentPostSectionLoading(config);
+  if (isDoc) await documentPostSectionLoading(getConfig());
```

Comment:
I still don't think this change is necessary. The result is the same... the object delivered from `getConfig()` is the config object.

Reply:
And if you're saying there's some property that doesn't exist yet, that says to me there's a race condition that should be addressed.

Reply:
@chrischrischris

Reply:
@auniverseaway I worked with @chrischrischris and he figured out the issue.  martech-metadata was using updateConfig (I figured since such a function existed, it would be the preferred way to update the config lol).  Turns out that function destroys the reference to the object, thereby not letting config see the updates I made.  So now that I'm no longer using setConfig in the block, I don't need to call getConfig in utils.